library-integration-slack/src/test/kotlin/library/integration/slack/services/error/handling/SlackErrorDecoderTest.kt

```diff
@@ -16,7 +16,7 @@ class SlackErrorDecoderTest {
 
     private val cut = SlackErrorDecoder()
 
-    private val headers = mapOf("Content-type" to arrayListOf("Applicatin/json"))
+    private val headers = mapOf("Content-type" to arrayListOf("application/json"))
```

Comment:
HTTP headers are handled case-insensitively but `Content-type` should be `Content-Type` just to be sure. You could also use Spring's `HttpHeaders.CONTENT_TYPE`. Similarly `application/json` can be got from Spring's `MediaType.APPLICATION_JSON_VALUE` constant.

Reply:
👍 Updated overall in the service.